pkg/registry/authentication/tokenreview/storage.go

```diff
@@ -53,6 +53,10 @@ func (r *REST) Create(ctx genericapirequest.Context, obj runtime.Object, include
 		return tokenReview, nil
 	}
 
+	if len(tokenReview.Spec.Token) == 0 {
```

Comment:
I would move this to just below the namespace check, before we check `tokenAuthenticator == nil`